.travis.yml

```diff
@@ -26,7 +22,7 @@ matrix:
   fast_finish: true
   include:
     - php: 5.4
-      env: COMPOSER_FLAGS="--prefer-lowest"
+      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER: weak
```

Comment:
your Yaml is invalid (because of the ``:`` here). This is why there is no Travis build